internal/core/manager.go

```diff
@@ -391,6 +391,11 @@ func (mngr *ArtifactManager) Push(options PushOptions) error {
 	mapUploadBlob := map[string]DiffRecord{}
 	for _, record := range result.Records {
 		if record.Type == DiffTypeAdd || record.Type == DiffTypeChange {
+			if record.Hash == "" {
```

Comment:
maybe we can encapsulate symlink check to a function in our later development (e.g. record.isSymlink())

Reply:
The record is DiffRecord. I think the `isSymbolic()` is bit confusing in different diff types.

Reply:
yeah, agree with you -> there is a DiffRecord context behind it.